code/game/objects/items/weapons/melee/misc.dm

```diff
@@ -53,3 +53,26 @@
 /obj/item/weapon/melee/chainsword/suicide_act(mob/user)
 	viewers(user) << "\red <b>[user] is slicing \himself apart with the [src.name]! It looks like \he's trying to commit suicide.</b>"
 	return (BRUTELOSS|OXYLOSS)
+
+
+//This is essientially a crowbar and a baseball bat in one.
+/obj/item/weapon/melee/misc/hammer
+	color

```

Comment:
No idea, I was wondering what was causing that to show up in the spawnable items list.